test/test_site/testLayouts.md

```diff
@@ -1,6 +1,7 @@
 <frontmatter>
 title: Test Layouts
 head: overwriteLayoutHead.md
+layout: testLayout
 </frontmatter>
 
-# Uses a layout
\ No newline at end of file
+# Uses a front matter layout
```

Comment:
In `site.json`, no layout is specified.
In the frontmatter, `testLayout` is specified.
The final output still uses `default` instead of `testLayout`.

That doesn't seem right. 😕

Reply:
My bad, fixed it, there was some issue with the glob layouts